website/sidebars.js

```diff
@@ -243,7 +243,7 @@ module.exports = {
                         'reference/public-signup',
                         'reference/projects',
                         'reference/project-collaboration-mode',
-                        'reference/provisioning',
+                        'reference/scim',
```

Comment:
would we need to add a redirect if someone has the old url?

Reply:
Don't see how they would have the old URL, so I don't think so, no. I did this, rather than patch the code so that all existing references would point to working docs